conf/messages.en

```diff
@@ -641,8 +641,8 @@ declaration.addDocument.remove.documentQuantity = Quantity of the measurement un
 supplementary.items = Items
 declaration.items = Items
 declaration.totalPackageQuantity = Total number of packages
-declaration.totalPackageQuantity.empty = Enter the total number of packages
-declaration.totalPackageQuantity.error = Enter the total number of packages
+declaration.totalPackageQuantity.empty = Enter the total number of packages up to 99999999
```

Comment:
Crazy.  It is effectively unlimited.  So why mention it?    
(I know it was asked for - just ranting)